libretroshare/src/services/autoproxy/p3i2pbob.cc

```diff
@@ -979,8 +936,8 @@ void p3I2pBob::finalizeSettings_locked()
 	sockaddr_storage_setipv4(mI2PProxyAddr, (sockaddr_in*)&proxy);
 	sockaddr_storage_setport(mI2PProxyAddr, 2827);
 
-	rslog(RsLog::Debug_Basic, &i2pBobLogInfo, "finalizeSettings_locked using " + sockaddr_storage_tostring(mI2PProxyAddr));
-	rslog(RsLog::Debug_Basic, &i2pBobLogInfo, "finalizeSettings_locked using " + mSetting.addr);
+	RS_DBG4("using ", sockaddr_storage_tostring(mI2PProxyAddr));
```

Comment:
AFAIR it would probably accept also `mI2PProxyAddr` and print it nicely without need of calling explicitly `sockaddr_storage_tostring`

Reply:
looks good:
> D 1591975806.104 void p3I2pBob::finalizeSettings_locked() using ipv4://127.0.0.1:2827